TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot.java

```diff
@@ -18,9 +20,11 @@ public static Robot getInstance() {
     }
 
     //add subsystems...
+    public Claw claw;
```

Comment:
add getClaw() 
```suggestion
    public Claw claw;
    
    public Claw getClaw (){ return this.claw; }
```

Reply:
This comment is correct, and should not have been resolved without fixing.
I will add that after creating a public method for getting the claw, the claw class member should be private.